pom.xml

```diff
@@ -74,6 +76,8 @@
     <google.protobuf.java.version>3.4.0</google.protobuf.java.version>
     <slf4j.version>1.7.5</slf4j.version>
     <spark.version>1.6.1</spark.version>
+    <hbase-shaded-server.version>1.4.10</hbase-shaded-server.version>
```

Comment:
keep the properties alphabetically sorted so that it's easier to find one.

Reply:
fixed